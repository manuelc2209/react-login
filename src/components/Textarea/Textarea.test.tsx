import { render, screen } from '@testing-library/react';

import { Textarea } from '.';

it('renders without crashing', () => {
    const { container } = render(<Textarea value="Test" onChange={vi.fn()} />);

    expect(container).toMatchSnapshot();
});

it('renders and finds the attribute value', () => {
    render(<Textarea value="Test" onChange={vi.fn()} />);

    expect(screen.getByText('Test')).toBeTruthy();
});
