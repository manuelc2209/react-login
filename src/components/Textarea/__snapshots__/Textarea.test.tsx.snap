// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders without crashing 1`] = `
<div>
  <textarea
    aria-label="Textarea"
    class="sc-bcXHqe cbnZNR"
  >
    Test
  </textarea>
</div>
`;
